snapcast-control/start.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+snapcastrd --port=80 --sc_host=localhost 
```

Comment:
To get the IP address of the current master server you can do:

```
  while ! curl -s "http://localhost:3000"; do sleep 1; done
  SNAPCAST_SERVER=$(curl --silent http://localhost:3000)
snapcastrd --port=80 --sc_host "$SNAPCAST_SERVER"
```

For the full example checkout: https://github.com/balenalabs/balena-sound/blob/master/snapcast-client/start.sh

Reply:
Also, after `fleet-supervisor` [Line27](https://github.com/balenalabs/balena-sound/blob/master/fleet-supervisor/server.js#L27) you'll need to restart this service with:

```
restartBalenaService('snapcast-control')
```

so each time the master server changes the `snapcast-control` finds out too.